typespec/Applications.Core/containers.tsp

```diff
@@ -322,6 +322,10 @@ enum VolumePermission {
 enum IAMKind {
   @doc("Azure Active Directory")
   azure,
+
+  // This was added because typespec doesn't generate necessary functions and other variables for single value enums.
+  // This can be deleted if we add more values to the enum.
+  string,
```

Comment:
Is there a tracking issue for this in the typespec repo we can refer to here?

Reply:
discussed. separate issue will be opened to update autorest version in radius project which may alleviate this issue.